Squablr/View Controllers/UserFeedViewController.m

```diff
@@ -113,15 +98,16 @@ - (void)swipeableView:(ZLSwipeableView *)swipeableView
 #pragma mark - ZLSwipeableViewDataSource
 
 - (UIView *)nextViewForSwipeableView:(ZLSwipeableView *)swipeableView {
-
-    CardView *view = [[CardView alloc] initWithFrame:swipeableView.bounds];
-    view.backgroundColor = [UIColor blueColor];
-    // Attempt to load user info into the card view
-    view.imageView.file = self.userProfilePhotos[0];
-    [view.imageView loadInBackground];
-    [self.userProfilePhotos removeObjectAtIndex:0];
-
-    return view;
+    // call the line below with the initwithProfile
+    if (self.profileIndex < self.userProfileInfo.count) {
+        // Access the profile array with the current profile index
+        Profile *currentProfile = self.userProfileInfo[self.profileIndex];
+        CardView *view = [[CardView alloc] initWithProfile:swipeableView.bounds profile:currentProfile];
```

Comment:
This constructor should be named
`[[CardView alloc] initWithBounds:swipeableView.bounds profile:currentProfile];`